libethereum/BlockChain.cpp

```diff
@@ -940,14 +940,17 @@ ImportRoute BlockChain::import(VerifiedBlockRef const& _block, OverlayDB const&
 	if (total.elapsed() > 0.5)
 	{
 		cnote << "SLOW IMPORT:" << _block.info.hash() << " #" << _block.info.number();
-		cnote << "  Import took:" << total.elapsed();
-		cnote << "  preliminaryChecks:" << preliminaryChecks;
-		cnote << "  enactment:" << enactment;
-		cnote << "  collation:" << collation;
-		cnote << "  writing:" << writing;
-		cnote << "  checkBest:" << checkBest;
-		cnote << "  " << _block.transactions.size() << " transactions";
-		cnote << "  " << _block.info.gasUsed() << " gas used";
+
+		unsigned const gasPerSecond = static_cast<double>(_block.info.gasUsed()) / enactment;
+		cnote << "  { \"importTime\": " << total.elapsed() << ", "
```

Comment:
Can you change it to a single line that starts with `SLOW IMPORT`, i.e. also move the block hash and number here?